python/CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Changelog for tchannel.py
 =========================
 
+0.9.2 (unreleased)
```

Comment:
@blampe you're wrong - when you are in 0.x every new feature, even if it is not backwards breaking, is a new minor. In fact, that is the same for 1.x